app/src/main/java/com/github/se/orator/ui/overview/SpeakingPracticeModule.kt

```diff
@@ -54,16 +55,12 @@ fun SpeakingPracticeModule(
   Scaffold(
       modifier = Modifier.fillMaxSize().testTag("speakingPracticeScreen"),
       topBar = {
-        // Use CenterAlignedTopAppBar for consistency
-        TopAppBar(
-            modifier =
-                Modifier.fillMaxWidth()
-                    .padding(top = AppDimensions.statusBarPadding)
-                    .testTag("topAppBar"),
+        TopNavigationMenu(
+            testTag = "topAppBar",
             title = {
               Text(
                   text = screenTitle,
-                  style = AppTypography.appBarTitleStyle.copy(fontWeight = FontWeight.Bold),
+                  style = AppTypography.mediumTopBarStyle,
                   color = MaterialTheme.colorScheme.onSurface,
                   modifier = Modifier.testTag("screenTitle"))
             },
```

Comment:
I think that you should remove this. #263 actually removed the titles from the speaking practice modules as the content was quite repetitive.

Reply:
this should be solved with the new changes